understanding/text-spacing.html

```diff
@@ -58,268 +58,199 @@
                </p>
-               <p>This SC focuses on the adaptability of content to an increase in spacing between lines,
-                  words, letters, and paragraphs. Any combination of these may assist a user with effectively
-                  reading text. As well, ensuring that content correctly adapts when users override
-                  author settings for spacing also significantly increases the likelihood other style
-                  preferences can be set by the user. For example, a user may need to change to a wider
-                  font family than the author has set in order to effectively read text. 
+               <p>この達成基準は、行、単語、文字及び段落間の間隔を広げることによるコンテンツの適応性に焦点を当てている。これらのすべての組み合わせによって、利用者がテキストを効率よく読むのに役立つことがある。また、利用者がコンテンツ制作者の間隔設定を上書きしたときにコンテンツが正しく適応するようにすることで、利用者が他のスタイルの設定を設定できる可能性も大幅に高まる。例えば、テキストを効果的に読むためには、利用者は、コンテンツ制作者が設定したものよりも幅の広いフォントファミリーに変更する必要があるかもしれない。
                </p>
                <section id="author-responsibility-">
                   
-                  <h3>Author Responsibility </h3>
+                  <h3>コンテンツ制作者の責任</h3>
                   
-                  <p>This SC does not dictate that authors must set all their content to the specified
-                     metrics, nor does the SC intend to imply that all users will adjust the specified
-                     metrics. Rather, it specifies that should a user choose to set any of these metrics
-                     they can do so without any loss of content or functionality. The author requirement
-                     is both to not interfere with a user's ability to override the author settings, and
-                     to ensure that content thus modified does not break content in the manners shown in
-                     figures 1 through 3 in Effects of Not Allowing for Spacing Override.
+                  <p>この達成基準は、コンテンツ制作者が全てのコンテンツを指定された指標に設定しなければならないことを指示するものではなく、又、全ての利用者が指定された指標に調整することを暗に示すものでもない。むしろ、利用者がこれらの指標のいずれかを設定することを選択した場合、コンテンツ又は機能を損なうことなく、そうすることができることを規定している。コンテンツ制作者の要件は、コンテンツ制作者の設定を利用者が上書きすることを妨げないことと、そのように変更されたコンテンツが、"間隔の上書きを許可しないことによる影響" の図 1～3 に示されている方法でコンテンツを壊させないようにすることの両方である。
                   </p>
                   
                   
-                  <p>In some human languages and scripts, some of the metrics specified by the SC are inapplicable.
-                     For example, languages such as Japanese do not use spacing following paragraphs, meaning
-                     that users are unlikely to make any paragraph spacing changes in practice. The exception
-                     in this SC allows authors to ignore text style properties which are inapplicable to
-                     the combination of language and script being used.
+                  <p>自然言語及び文字体系の中には、達成基準が規定する指標の一部が適用できないものがある。例えば、日本語のような言語は段落に続く間隔を用いないため、利用者が実際に段落の間隔を変更する可能性は低い。この達成基準の例外は、使用する言語及び文字体系の組み合わせに適用できないテキストのスタイルプロパティについては無視することをコンテンツ制作者に許容している。
```

Comment:
「AとBとの組み合わせ」かなと。
「スタイルプロパティについては無視する」はちょっと苦しいですかね… allow A to doは「Aが…できるようにする」とすれば「スタイルプロパティをコンテンツ制作者が無視できるようにする」が自然でしょうか…。

```suggestion
                  <p>自然言語及び文字体系の中には、達成基準が規定する指標の一部が適用できないものがある。例えば、日本語のような言語は段落に続く間隔を用いないため、利用者が実際に段落の間隔を変更する可能性は低い。この達成基準の例外は、使用する言語と文字体系との組み合わせに適用できないテキストのスタイルプロパティをコンテンツ制作者が無視できるようにする。
```